cfp.yml

```diff
@@ -0,0 +1,21 @@
+"PLDI'25":
+  - url: "[PLDI'25](https://conf.researchr.org/series/pldi)"
+  - publisher: ACM
+  - CORE: "[A*](https://portal.core.edu.au/conf-ranks/84/)"
+  - Scope: PL
+  - Short pages:
+  - Full pages: 20
+  - Format: 1C
+  - CFP:
+  - Country: Denmark
+
+"POPL'25":
+  - url: "[POPL'25](https://conf.researchr.org/home/POPL-2025)"
+  - publisher: ACM
+  - CORE: "[A*](https://portal.core.edu.au/conf-ranks/82/)"
+  - Scope: SE
+  - Short pages:
+  - Full pages: 25
+  - Format:
+  - CFP: 11.07.2025
```

Comment:
@uchitsa maybe it's better to use ISO format: `2025-07-11` (YYYY-MM-DD)

Reply:
updated